providers/trello.js

```diff
@@ -4,16 +4,35 @@
 
 const baselink = 'https://trello.com/';
 const avatarurl = 'https://trello-avatars.s3.amazonaws.com/';
+const colors = {
+    blue: 31167,      // #0079bf
+    orange: 13799476, // #d29034
+    green: 5347385,   // #519839
+    red: 11552306,    // #b04632
+    purple: 9003166,  // #89609e
+    pink: 13458065,   // #cd5a91
+    lime: 4964203,    // #4bbf6b
+    sky: 44748,       // #00aecc
+    grey: 8621201     // #838c91
```

Comment:
It is maybe easier to directly write the integer numbers here, but for easier color handling wouldn't it be better to write the hex codes in there?

Just Like:
blue: 0x0079BF